src/pages/en/tutorial/0-introduction/1.md

```diff
@@ -15,7 +15,7 @@ If you have some basic familiarity with **HTML**, **Markdown**, **CSS**, and a l
 
 Yes! This tutorial is for everyone!
 
-**Newer to web dev?** You can use [Unit 1](/en/tutorial/1-setup/) to make sure you have the development tools and online accounts you'll need to complete the tutorial and provides instructions for using a local setup with VS Code or a browser-based Stackblitz setup. It will walk you through creating a new Astro project, storing it on GitHub and deploying to Netlify.
+**Newer to web dev?** You can use [Unit 1](/en/tutorial/1-setup/) to make sure you have the development tools and online accounts you'll need to complete the tutorial, using a local setup with VS Code, or a browser-based setup with Stackblitz. It will walk you through creating a new Astro project, storing it on GitHub and deploying to Netlify.
```

Comment:
```suggestion
**Newer to web dev?** You can use [Unit 1](/en/tutorial/1-setup/) to make sure you have the development tools and online accounts you'll need to complete the tutorial. It will walk you through creating a new Astro project, storing it on GitHub and deploying to Netlify.
```
We're actually planning to make some further edits to this page tomorrow, but in the meantime, taking this out makes the sentence even easier to read.